src/main/java/qupath/ext/instanseg/core/InstanSegOutputToObjectConverter.java

```diff
@@ -14,21 +16,17 @@
 import qupath.lib.objects.PathObject;
 import qupath.lib.objects.PathObjects;
 import qupath.lib.objects.PathTileObject;
+import qupath.lib.objects.classes.PathClass;
 import qupath.lib.regions.ImagePlane;
 import qupath.lib.roi.GeometryTools;
 import qupath.lib.roi.interfaces.ROI;
 import qupath.opencv.tools.OpenCVTools;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Random;
+import java.util.*;
```

Comment:
Trying to convince IntelliJ now to do this...

Reply:
Yes, I swear I changed the setting the other day...